src/pycrop2ml_ui/menus/writeXML/writeunitxml.py

```diff
@@ -111,7 +111,7 @@ def _createInit(self):
 
     def _createAlgo(self):
```

Comment:
You parse the model before creating algo file.
You need to create an empty algo file when you create the model. So you can parse the model and update algo file with model documentation.